roles/satellite-clone/tasks/main.yml

```diff
@@ -1,10 +1,15 @@
 ---
-- include_vars: main.yml
-
 # include pre_install_check
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: "check that variables are updated"
+  fail: msg="Please update the variables in satellite-clone-vars.yml (after copying satellite-clone-vars.sample.yml)"
+  when:
+    - hostname == "satellite.example.com"
+    - activationkey == "changeme"
+    - org == "111111"
```

Comment:
@ehelms my thought was that if they changed at least one, that confirms that they copied the local vars file over and edited it. But an "and" operator could be a more thorough check, provided a user wouldn't want to use a default (like satellite.example.com)